geomesa-blobstore/geomesa-blobstore-core/src/main/scala/org/locationtech/geomesa/blob/core/AccumuloBlobStore.scala

```diff
@@ -78,6 +79,32 @@ class AccumuloBlobStore(ds: AccumuloDataStore) {
     }
   }
 
+  def delete(id: String): Unit = {
+    // TODO: Get Authorizations using AuthorizationsProvider interface
+    // https://geomesa.atlassian.net/browse/GEOMESA-986
+    val bd = connector.createBatchDeleter(blobTableName, new Authorizations(), bwc.getMaxWriteThreads, bwc)
+    bd.setRanges(List(new Range(new Text(id))))
+    bd.delete()
+    bd.close()
+    deleteFeature(id)
+  }
+
+  private def deleteFeature(id: String): Unit = {
+    val transaction = new DefaultTransaction("removeBlobFeature")
```

Comment:
I think I followed a unit test in accumulofeaturewriter, would switching to autocommit make this just work?